docs/4.0. Behaviour.md

```diff
@@ -230,29 +230,30 @@ Each activation is allocated a unique identifier by the API. This identifier is
 
 Note the presence of the extra `activation_time` parameter in the response schema. For immediate activation this SHOULD be the time the activation actually occurred as an absolute TAI timestamp.
 
-An HTTP 200 response SHOULD be returned when an immediate activation has been successfully requested. For a request with an immediate activation the API SHOULD only return a response once the new map has been applied to the audio.
+An HTTP 200 response MUST be returned when an immediate activation has been successfully requested. For a request with an immediate activation the API SHOULD only return a response once the new map has been applied to the audio.
 
-An HTTP 202 response SHOULD be returned when a request for a scheduled activation is accepted, to indicate that while the request itself was acceptable it has not yet been acted upon by the sender. For scheduled activations `activation_time` should be the absolute TAI time the parameters will actually transition. For absolute scheduled activations SHOULD be the same as the requested time, but may differ if the Device is unable to schedule at the requested time.
+An HTTP 202 response MUST be returned when a request for a scheduled activation is accepted, to indicate that while the request itself was acceptable it has not yet been acted upon by the sender. For scheduled activations `activation_time` SHOULD be the absolute TAI time the parameters will actually transition. For absolute scheduled activations SHOULD be the same as the requested time, but may differ if the Device is unable to schedule at the requested time.
 
-If an Output is modified by an already scheduled activation the API SHOULD respond with HTTP 423 `resource locked`. The API MUST reject the entire request, even if some of the routes in the failed request are not currently locked by another activation.
+In there is an error in the request, the API MUST reject the entire request with an appropriate 4xx response. Particular responses MUST be used in these cases:
```

Comment:
Can more than one of these cases apply at the same time? Does that warrant any extra wording?

Reply:
We think it's ok.